python/ql/lib/semmle/python/Concepts.qll

```diff
@@ -443,38 +443,41 @@ module RegexExecution {
   }
 }
 
-/**
- * A data-flow node that executes an LDAP query.
- *
- * Extend this class to refine existing API models. If you want to model new APIs,
- * extend `LDAPQuery::Range` instead.
- */
-class LdapExecution extends DataFlow::Node {
-  LdapExecution::Range range;
-
-  LdapExecution() { this = range }
-
-  /** Gets the argument containing the filter string. */
-  DataFlow::Node getFilter() { result = range.getFilter() }
-
-  /** Gets the argument containing the base DN. */
-  DataFlow::Node getBaseDn() { result = range.getBaseDn() }
-}
-
-/** Provides classes for modeling new LDAP query execution-related APIs. */
-module LdapExecution {
+/** Provides classes for modeling LDAP-related APIs. */
+module LDAP {
```

Comment:
nitpick: our style guide suggests that because this acronym is longer than 2 characters, it should be written as `Ldap`. I guess there is some value in having consistency between the existing `HTTP` module. I'm not really sure whether this is something we need to change...

Reply:
Yeah, I modelled it after the HTTP module. I guess we can change both in a clean-up PR? Or are we creating deprecation work for ourselves that way?

Reply:
I think it's OK